source/dem/force_chains_visualization.cc

```diff
@@ -527,7 +527,7 @@ ParticlesForceChains<dim, contact_model, rolling_friction_model>::
 
   for (unsigned int i = 0; i < n_files; ++i)
     filenames.push_back(file_prefix + "." + Utilities::int_to_string(iter, 5) +
-                        "." + Utilities::int_to_string(i, 5) + ".vtu");
+                        ".vtu");
```

Comment:
It still does work for me, I think this is the line 51 that you need to fix.
Look at the lines 62-67 in the `solutions_output.cc`, the filename should be handled the same way. :)
I guess there is a reason, but why don't you call the `write_vtu_and_pvd()` in your function? I'm wondering if the .pvd are actually being modified at each core here, which is avoid in the basic `write_vtu_and_pvd()`.

Reply:
It was working fine this morning, sorry for that. I'm gonna double check everything again.

Reply:
Also, do you open the .pvd in Paraview? If not, it will be better to test if it works :) 
Also, you can open the file .pvtu and .pvd as text files to see how this is encoded. 
All the binary information are in the .vtu, the .pvtu is it the parent of the .vtu (you can have .vtu on many core for the same iteration and the .pvtu will linked them, the extra 5 digits of the .vtu are the core id, Bruno correct me on that one if I'm wrong since we don't use that in DEM) and the .pvd will contains the links/pointers to .pvtu.
If it helps x)

Reply:
Btw, maybe it is working, I forgot to recompile lol. But if you wanna keep the same filenames was other output files, the comment is still relevant

Reply:
Yep, I did all that. Pvd files was working in paraview and the names was the same in pvtu and vtu files. Maybe I did a mishandling when i commit, pull and/or push. I'll make another commit with those mistakes removed.

Reply:
It is working, sorry. If you perfer to keep the filename as is, you can, just need to remove the loop since you no longer needs the i variable

Reply:
Perfect! Thank you Audrey :))